docs/connect/odbc/linux-mac/install-microsoft-odbc-driver-sql-server-macos.md

```diff
@@ -53,7 +54,7 @@ HOMEBREW_NO_ENV_FILTERING=1 ACCEPT_EULA=Y brew install msodbcsql17 mssql-tools
 > If you installed the v17 `msodbcsql` package that was briefly available, you should remove it before installing the `msodbcsql17` package. This will avoid conflicts. The `msodbcsql17` package can be installed side by side with the `msodbcsql` v13 package.
 
 
-## <a id="13.1"></a> ODBC 13.1
+### <a id="13.1"></a> ODBC 13.1
```

Comment:
Revert. We want the two previous versions to be available via the right-side context links.